### PR TITLE
DM-30741: Add a job id event to the OCPS.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: sal_interfaces/OCPS/OCPS_Events.xml [hunk 1]
```diff
@@ -28,4 +28,31 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>OCPS</Subsystem>
+    <EFDB_Topic>OCPS_logevent_job_id</EFDB_Topic>
```
Comment (mareuter) [inline sal_interfaces/OCPS/OCPS_Events.xml:33]: I have a preference for this being called: OCPS_logevent_jobId.

File: sal_interfaces/OCPS/OCPS_Events.xml [hunk 2]
```diff
@@ -28,4 +28,31 @@
...
+    <item>
+      <EFDB_Name>event_name</EFDB_Name>
```
Comment (mareuter) [inline sal_interfaces/OCPS/OCPS_Events.xml:35]: I think in order to maintain consistency in the attributes either write them as:
- eventName
- seqNum
- jobId

or
- event_name
- seq_num
- job_id

**Review by mareuter — APPROVED** [review]
A couple of naming suggestions, but otherwise this is fine.

File: sal_interfaces/OCPS/OCPS_Events.xml (hunk 1 above)
Comment (ktlim) [reply sal_interfaces/OCPS/OCPS_Events.xml:33]: The name `job_id` was for parallelism with the existing `job_result`.

File: sal_interfaces/OCPS/OCPS_Events.xml (hunk 2 above)
Comment (ktlim) [reply sal_interfaces/OCPS/OCPS_Events.xml:35]: Similarly, the field in `job_result` is `job_id`.  `seqNum` was for parallelism with its underlying source: `private_seqNum`, as that is an important detail needed for its usage.  But if we don't mind usage like `event.seq_num == trigger.private_seqNum`, I'm happy to change it.

Comment (mareuter) [reply sal_interfaces/OCPS/OCPS_Events.xml:35]: OK. If there is precedent for they way things are names, then they can all stay the way you originally wrote them.

**Pull request merged**